src/frankenphp-symfony/src/Runner.php

```diff
@@ -22,22 +22,25 @@ public function __construct(private HttpKernelInterface $kernel)
 
     public function run(): int
     {
+        $kernel = $this->kernel;
```

Comment:
Hmm, `static function` might not be worth it here since it has a reason to use `$this`.

Reply:
Yes I was hesitating but let's keep this simple indeed.